yml/hsc.yaml

```diff
@@ -3985,3 +3985,8 @@ tables:
     datatype: boolean
     description:
     mysql:datatype: BOOLEAN
+  - name: detect_isPrimary
+    "@id": "#Source.detect_isPrimary"
+    datatype: boolean
+    description:
```

Comment:
I am sad that all of the `description` fields in this 3900-line yaml situation are empty. I suppose you've followed convention here, though, so I won't let it stop you.